en/tips/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+# General troubleshooting
+
+Follow these steps to solve your issue as fast as possible.
+
+## My Mautic is acting veird
+
+Even through the effort of the dev and test teams, it might happen. At first, let's try a few tricks which might fix it fast.
+
+### 1. Clear the cache
+
+There are several ways how to do that. The easiest is to go to the `/app/logs` folder and delete its content. If you want to do it via CLI command, navigate to the Mautic root folder and run `rm -rf app/cache/*`. The new cache files will generate itself after next Mautic refresh in the browser.
+
+It might happen that the files won't generate itself. It can be caused by the wrong folder permission and Mautic doesn't have permission to write the new cache files. Contact your sysadmin and ask him to fix it for you.
+
+Tip: Don't execute Mautic commands as the root user yourself and do not run Mautic commands in the root crontab. This way all the files which will be created by the command will have the root as the author and Mautic won't be able to rewrite those files.
+
+### 2. Try to fix the database schema
+
+Go to `http://[your-mautic-url]/s/update/schema`. It will update your database schema if there are some known updates to be applied.
+
+## Nope, it didn't help
+
+Alright. Let's figure out what the issue really is.
+
+### Check the logs
+
+#### Mautic logs
+
+Please, do not create a new issue/post/comment anywhere where you'll post that the error message is:
```

Comment:
Maybe change this to something like "Please do not report the following two errors:"